source/_integrations/ohme.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Ohme
+description: Instructions to configure the Ohme integration into Home Assistant.
+ha_category:
+  - Sensor
+  - Car
+ha_release: 2025.1
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@dan-r'
+ha_config_flow: true
+ha_domain: ohme
+ha_platforms:
+  - sensor
+  - button
+---
+
+The **Ohme** {% term integration %} allows you to connect your [Ohme](https://ohme-ev.com/) EV charger to Home Assistant.
+
+
+## Prerequisites
+
+- A compatible charger. This integration has been tested with:
+  - Ohme Home Pro
+  - Ohme Home
+  - Ohme Go
+  - Ohme ePod
```

Comment:
This should be a supported device section. For reference:

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-devices